distribution/config/nginx.conf

```diff
@@ -29,6 +29,8 @@ server {
             # Jenkins environment is booting
             return 521;
         }
+        proxy_set_header Host $host:$server_port;
+        proxy_redirect http://localhost/ http://localhost:8080/;
```

Comment:
`8080` is correct wrt. our current documentation, but is not if user didn't use `-p 8080:80` as documented right?